### PR TITLE
Configuration tab, conditions being sent to server

### PR DESCRIPTION
**Pull request by rpiaggio**
This PR sets up a proxy to communicate with the web server in dev mode.

Furthermore, changing conditions is actually invoking the server.

### CODE REVIEW

File: modules/web/client/src/main/publicdev/environments.conf.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   {
     "hostName": "*",
     "environment": "DEVELOPMENT",
-    "odbURI": "wss://lucuma-postgres-odb-dev.herokuapp.com/ws",
+    "odbURI": "ws://localhost:8082/ws",
```
Comment (cquiroz) [inline modules/web/client/src/main/publicdev/environments.conf.json:5]: Do we need to run a local. odb?

**Review by cquiroz — APPROVED** [review]
LGTM, though I'm not sure if I want to have the default odb as localhost

File: modules/web/client/src/main/publicdev/environments.conf.json (hunk 1 above)
Comment (rpiaggio) [reply modules/web/client/src/main/publicdev/environments.conf.json:5]: This slipped through, I'll revert. We have logic in place to pick up `local.conf.json` if need be.

Comment (rpiaggio) [reply modules/web/client/src/main/publicdev/environments.conf.json:5]: Actually, this shouldn't be checked in, it got out of sync with `.gitignore` with the directory change.

**Pull request merged**